runners/console/index.ts

```diff
@@ -353,6 +405,13 @@ export class Console extends Runner {
         return result;
     }
 
+    runChangeWorkspace(runCommand: RunCommand): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+        this.setVariable(this.workspaceDirectory, path.join(this.getWorkingDirectory(), runCommand.command.parameters[0]));
```

Comment:
I think it would be good to check if the given directory exist and to create it if not.
You should also add a assert method for changeWorkspace. Just to keep it equal with the other methods.